lib/index.js

```diff
@@ -28,11 +28,12 @@ function setStatementTimeout(client) {
     }
   });
 }
-function stringifyProtobuf(layers, tile) {
+
+// allowing tho options object for addGeoJson() - simplification settings, etc.
+function stringifyProtobuf(layers, tile, mvtOptions) {
   var vtile = new mapnik.VectorTile(tile.z, tile.x, tile.y);
   for (var layerName in layers) {
-    vtile.addGeoJSON(JSON.stringify(layers[layerName]), layerName);
-
+    vtile.addGeoJSON(JSON.stringify(layers[layerName]), layerName, mvtOptions);
```

Comment:
the [addGeoJSON](http://mapnik.org/documentation/node-mapnik/3.5/#VectorTile.addGeoJSON) method is where we can influence simplification. Or any other options for that matter.